src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -2168,10 +2174,23 @@ private ResponseEntity<SubmittedCallbackResponse> servePacksWithConfidentialDeta
 
         caseDataMap.put(FINAL_SERVED_APPLICATION_DETAILS_LIST, finalServedApplicationDetailsList);
 
+        if (caseData.getServiceOfApplication().getSoaServeToRespondentOptions() != null
+            && YesOrNo.No.equals(caseData.getServiceOfApplication().getSoaServeToRespondentOptions())) {
```

Comment:
Is this valid? I guess this field will be cleared after SOA event and soaServeToRespondentOptions will be null at this moment